cmake/inference_lib.cmake

```diff
@@ -16,8 +16,8 @@
 set(FLUID_INSTALL_DIR "${CMAKE_BINARY_DIR}/fluid_install_dir" CACHE STRING
```

Comment:
is FLUID_INSTALL_DIR should be changed too?

Reply:
FLUID_INSTALL_DIR seems to be the total product of paddle compilation, do we need to change it to PADDLE_INSTALL_DIR ?

Reply:
I think so, otherwise it will be strange. ${CMAKE_BINARY_DIR}/paddle_install_dir  and ${CMAKE_BINARY_DIR}/paddle_inference_install_dir

Reply:
done